README.md

```diff
@@ -1,4 +1,255 @@
+```
+
+## Start CockroachDB
+
+Download [`example.yaml`](https://github.com/cockroachdb/cockroach-operator/examples/example.yaml) and optionally modify this configuration as below.
+
+```
+curl -OOOOOOOOO https://raw.githubusercontent.com/cockroachdb/cockroach-operator/master/examples/example.yaml
+```
+
+By default `example.yaml` tells the Operator to create a secure cluster. For local testing, you can optionally deploy an insecure cluster. Exclude the following line from the configuration and skip the [Certificate signing](#certificate-signing) section below.
+
+```
+  tlsEnabled: true
+```
+
+An insecure cluster is suitable for *testing purposes only* and should not be used on production.
+
+### Certificate signing
+
+By default, the Operator uses the built-in Kubernetes CA to generate and approve 1 client and 1 node certificate for the cluster.
+
+> This differs from the way CockroachDB handles [node authentication](https://www.cockroachlabs.com/docs/stable/authentication.html#using-digital-certificates-with-cockroachdb), in which each CockroachDB node has its own node certificate. Using the Operator, one node certificate is currently used to authenticate the cluster.
+
+<!-- If you wish to use a non-Kubernetes CA, additionally enable the following lines in `example.yaml` and fill in the names of the corresponding node and client secrets:
+
+```
+nodeTLSSecret: <node-secret>
+clientTLSSecret: <client-secret>
+```
+
+For example, if you are using [`cockroach cert`](https://www.cockroachlabs.com/docs/stable/cockroach-cert.html) to generate certificates, use these secret names:
+
+```
+nodeTLSSecret: cockroachdb.node
+clientTLSSecret: cockroachdb.client.root
+``` -->
+
+### Set resource requests and limits
+
+Before deploying, we recommend explicitly allocating CPU and memory to CockroachDB in the Kubernetes pods. This is done in the `resources.requests` object in `example.yaml`. For example:
+
+```
+resources:
+  requests:
+    cpu: "16"
+    memory: "8Gi"
+  limits:
+    memory: "8Gi"
+```
+
+### Create the StatefulSet
+
+Apply `example.yaml`:
+
+```
+kubectl create -f example.yaml
+```
+
+Check that the pods were created:
+
+```
+kubectl get pods
+```
+
+```
+NAME                                  READY   STATUS    RESTARTS   AGE
+cockroach-operator-6f7b86ffc4-9t9zb   1/1     Running   0          3m22s
+cockroachdb-0                         1/1     Running   0          2m31s
+cockroachdb-1                         1/1     Running   1          102s
+cockroachdb-2                         1/1     Running   0          46s
+```
+
+Each pod should have `READY` status soon after being created.
+
+## Access the SQL shell
+
+Launch a pod that runs indefinitely with the `cockroach` binary. <!-- Depending on how you authenticated the cluster: -->
+
+- Kubernetes CA:
+
+	```
+	kubectl create -f https://raw.githubusercontent.com/cockroachdb/cockroach-operator/master/examples/client-secure.yaml
```

Comment:
This does not work out of the box.  We do not need the init-container that is in this example.

Reply:
I'm pretty sure this worked for me in testing. Was there a change that breaks this config?